packages/thirdweb/src/v2/types.ts

```diff
@@ -0,0 +1 @@
+export type EthereumAddress = `0x${string}`; // Joaquim do you think we should use the types from the existing SDK or create entirely new ones so we don't cross over at all?
```

Comment:
i think if we keep things namespaced we can get away with new types.

but tbh for addresses i still fill like 0x{string} is overkill. super annoying when you get back nested structs from api that you're just trying to feed back into a typed function. I would just use string, and validate at runtime